<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTP_clientOptions_Dialog</class>
 <widget class="QDialog" name="FTP_clientOptions_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTP Client Options</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Server IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IP_lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1234</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="btr_frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>291</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bitrate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bitrate_lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="userbitrate_checkBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Use realtime data transfer</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
